src/maud/utils.py

```diff
@@ -53,3 +54,15 @@ def get_normal_parameters_from_quantiles(x1, p1, x2, p2):
     sigma = (x2 - x1) / denom
     mu = (x1 * norm.ppf(p2) - x2 * norm.ppf(p1)) / denom
     return mu, sigma
+
+
+def null(a, rtol=1e-5):
```

Comment:
I would rename to `get_null_space` - this maintains the convention of function names being verbs and avoids having something called 'null' in our namespace.